subscriptions.md

```diff
@@ -0,0 +1,304 @@
+# Subscriptions Endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint contains all of the subscription requested
+
+## Main Endpoint
+**GET /api/core/subscriptions**
+
+Retrieve a pageable list of subscriptions. 
+The subscriptions are ordered by eperson number ascending.
+The JSON response document is as follow.
+
+```json
+{
+  "_embedded" : {
+    "subscriptions" : [ {
+      "id" : 1,
+      "type" : "subscription",
+      "subscriptionParameterList" : [ {
+        "name" : "frequency",
+        "value" : "D"
+      } ],
+      "subscriptionType" : "content",
+      "_links" : {
+        "resource" : {
+          "href" : "http://localhost/api/core/subscriptions/1/resource"
+        },
+        "eperson" : {
+          "href" : "http://localhost/api/core/subscriptions/1/eperson"
+        },
+        "self" : {
+          "href" : "http://localhost/api/core/subscriptions/1"
+        }
+      }
+    } ]
+  },
+  "_links" : {
+    "self" : {
+      "href" : "http://localhost/api/core/subscriptions"
+    },
+    "search" : {
+      "href" : "http://localhost/api/core/subscriptions/search"
+    }
+  },
+  "page" : {
+    "size" : 20,
+    "totalElements" : 1,
+    "totalPages" : 1,
+    "number" : 0
+  }
+}
+```
+
+Attributes
+* page, size [see pagination](README.md#Pagination)
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated.
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only administrators can access the endpoint
+
+## Single Subscription Object
+**GET /api/core/subscription/<:id>**
+
+Provide detailed information about a specific subscription.
+The JSON response document is as follow
+
+```
+{
+  "id" : 1,
+  "type" : "subscription",
+  "subscriptionParameterList" : [ {
+    "name" : "frequency",
+    "value" : "W"
+  } ],
+  "subscriptionType" : "content",
+  "_links" : {
+    "resource" : {
+      "href" : "http://localhost/api/core/subscriptions/1/resource"
+    },
+    "eperson" : {
+      "href" : "http://localhost/api/core/subscriptions/1/eperson"
+    },
+    "self" : {
+      "href" : "http://localhost/api/core/subscriptions/1"
+    }
+  }
+}
+```
+
```

Comment:
please describes the properties of the json, more specifically 
- the subscriptionParamterList where the "frequency" is the currently only supported name and the different values (D, W, M). 
- subscriptionType: currently only content but open to future extensions like statistics